src/utils/records.js

```diff
@@ -15,12 +29,22 @@ export function validateRecord(record) {
     case 'address':
       return addressUtils.isAddress(value)
     case 'content':
-      return validateContent(value)
+      return validateContent(value) // todo wil this ever be reached? if type is 'content', the above if statement will be triggered
```

Comment:
`if (type == 'content' && record.contentType === 'oldcontent') {` only triggers if `contentType` is also 'oldcontent` which is not always the true for old `content` types